docs/UserGuide.md

```diff
@@ -131,19 +143,35 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by phone number: `findnum`
+
+Finds persons whose phone number matches the given keywords.
+
+Format: `find PHONE_NUMBER`
+
+* The search is strictly numeric. e.g `83647382` will only match `83647382`.
+* The order of the keywords matters. e.g. `83647382` will not match `83467328`
+* Only the full number is searched. eg. `83647382` will not match `83641001`
+
+Examples:
+* `find 83647382` returns `john` and `John Doe`
+* `find 91739272` returns `David Li`<br>
+
 ### Deleting a person : `delete`
 
 Deletes the specified person from the address book.
 
-Format: `delete INDEX`
+Format: `delete [n/NAME] i/ID`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the person with the specified `ID`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Feature spec v1.2 might be referring to UID instead of the index number shown in displayed person list. Require @purivirakarin comments.